diff --git a/package/libglib2/libglib2.mk b/package/libglib2/libglib2.mk
index fda0433..2c0eead 100644
--- a/package/libglib2/libglib2.mk
+++ b/package/libglib2/libglib2.mk
@@ -84,6 +84,8 @@ define LIBGLIB2_REMOVE_GDB_FILES
 endef
 endif

+LIBGLIB2_CONF_OPTS += --default-library both
+
 ifeq ($(BR2_PACKAGE_UTIL_LINUX_LIBMOUNT),y)
 LIBGLIB2_CONF_OPTS += -Dlibmount=enabled
 LIBGLIB2_DEPENDENCIES += util-linux
